pkg/csi_driver/multishare_ops_manager_test.go

```diff
@@ -573,6 +622,253 @@ func TestListInstanceForStorageClassPrefix(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "non-specified sc prefix in csi req",
+			req: &csi.CreateVolumeRequest{
+				Parameters: map[string]string{},
+			},
+			target: &file.MultishareInstance{
```

Comment:
target instance should always contain a prefix. this is not a valid test case. check getInstanceSCPrefix() function

Reply:
Done